### PR TITLE
update o365

### PR DESCRIPTION
**Pull request by rombernier**
https://github.com/SekoiaLab/platform/issues/50756

### CODE REVIEW

File: Office 365/o365/_meta/fields.yml [hunk 1]
```diff
@@ -60,6 +60,26 @@ email.to.address:
...
 
+browser.name:
+  description: ""
+  name: browser.name
+  type: keyword
+
+browser.version:
+  description: ""
+  name: browser.version
+  type: keyword
```
Comment (squioc) [inline Office 365/o365/_meta/fields.yml:71]: Not ECS fields. Must be prefixed.

**Review by squioc — CHANGES_REQUESTED** [review]

File: Office 365/o365/_meta/fields.yml (hunk 1 above)
Comment (rombernier) [reply Office 365/o365/_meta/fields.yml]: Done, thanks

File: Office 365/o365/ingest/parser.yml [hunk 2]
```diff
@@ -422,6 +426,18 @@ stages:
...
+      - set:
+          office365.source.application: "{{json_event.message.SourceApp}}"
+          office365.form_name: "{{json_event.message.FormName}}"
+
+  parse_browser:
+    actions:
+      - set:
+          office365.browser.name: "{{json_event.message.BrowserName}}"
+          office365.browser.version: "{{json_event.message.BrowserVersion}}"
+
```
Comment (squioc) [inline Office 365/o365/ingest/parser.yml:440]:
```suggestion
          user_agent.name: "{{json_event.message.BrowserName}}"
          user_agent.version: "{{json_event.message.BrowserVersion}}"

```

File: Office 365/o365/_meta/fields.yml [hunk 3]
```diff
@@ -60,6 +60,26 @@ email.to.address:
...
+office365.browser.name:
+  description: ""
+  name: office365.browser.name
+  type: keyword
+
+office365.browser.version:
+  description: ""
+  name: office365.browser.version
+  type: keyword
+
```
Comment (squioc) [inline Office 365/o365/_meta/fields.yml:72]: My bad. These two fields can be replaced with `user_agent.*`:
```suggestion
```

**Review by squioc — CHANGES_REQUESTED** [review]

File: Office 365/o365/_meta/fields.yml (hunk 3 above)
Comment (rombernier) [reply Office 365/o365/_meta/fields.yml]: Ok, yes, I fix that

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**